packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -70,11 +71,14 @@ export class GroupHeaderBase extends BaseComponent<IGroupHeaderProps, IGroupHead
     const isSelectionCheckVisible = canSelectGroup && (isCollapsedGroupSelectVisible || !(group && group.isCollapsed));
     const currentlySelected = isSelected || selected;
 
+    const isRTL = getRTL();
+
     this._classNames = getClassNames(styles, {
       theme: theme!,
       className,
       selected: currentlySelected,
-      isCollapsed
+      isCollapsed,
+      compact: compact
```

Comment:
nit: could just be "compact" by itself like the others

Reply:
Fixed.